core/server/api/posts.js

```diff
@@ -12,6 +12,7 @@ var Promise = require('bluebird'),
         'created_by', 'updated_by', 'published_by', 'author', 'tags', 'fields',
         'next', 'previous', 'next.author', 'next.tags', 'previous.author', 'previous.tags'
     ],
+    unsafeAttrs = ['author', 'author_id'],
```

Comment:
I don't think we need the property `author`, because of https://github.com/TryGhost/Ghost/blob/master/core/server/api/utils.js#L300?